lib/ansible/runner/credential/rhv_credential.rb

```diff
@@ -0,0 +1,49 @@
+module Ansible
+  class Runner
+    class RhvCredential < Credential
+      def self.auth_type
+        "ManageIQ::Providers::EmbeddedAnsible::AutomationManager::RhvCredential"
+      end
+
+      # Modeled off of rhv injectors for awx:
+      #
+      #   https://github.com/ansible/awx/blob/1242ee2b/awx/main/models/credential/__init__.py#L1035-L1054
+      #
+      def env_vars
+        write_credentials_file
+        {
+          "OVIRT_INI_PATH" => rhv_credentials_file,
+          "OVIRT_URL"      => auth.host || "",
+          "OVIRT_USERNAME" => auth.userid || "",
+          "OVIRT_PASSWORD" => auth.password || "",
+        }
+      end
+
+      def write_credentials_file
+        rhv_data = %W[
```

Comment:
Why not a heredoc vs array+join?

Reply:
Because of the comment below.  If I want to add something to it, it was easier to do with an array.

I guess I could have just added a new line as well, but ¯\\(°\_o)/¯